add_undercloud_to_inventory.yml

```diff
@@ -16,6 +16,24 @@
         ansible_user: "stack"
         ansible_ssh_private_key_file: "{{ ansible_ssh_key }}"
         ansible_python_interpreter: "{{ python_interpreter }}"
+
+    - name: add localhost to inventory file
+      add_host:
+        name: "localhost"
+        groups: "local"
+        ansible_user: "root"
+        ansible_python_interpreter: "{{ python_interpreter }}"
+
+    - name: add freeipa_host to inventory file
+      add_host:
+        name: "freeipa"
+        groups: "freeipa"
+        ansible_host: "{{ freeipa_hostname }}"
```

Comment:
How is " freeipa.redhat.local" which is freeipa_hostname resolved?

Reply:
The hostname is changed to freeipa.redhat.local during the undercloud deployment and is copied into the /etc/hosts to resolve it